tools/integration_tests/util/setup/setup.go

```diff
@@ -269,6 +269,14 @@ func IgnoreTestIfIntegrationTestFlagIsSet(t *testing.T) {
 	}
 }
 
+func IgnoreTestIfIntegrationTestFlagNotIsSet(t *testing.T) {
```

Comment:
nit: IgnoreTestIfIntegrationTestFlagIsNotSet

Also can you please write comment above that this function helps in avoid running integration tests in unit tests.